admin/lib/errors/pins-overview.js

```diff
@@ -0,0 +1,7 @@
+'use strict'
+
+module.exports = {
+  'noCurrentCheckWindow': 'SM01. Please contact helpdesk',
```

Comment:
Possibly it might be easier if all the error codes where in a single file - we will need to provide documentation on every single one of them.

In fact - perhaps the documentation should be a part of the PR that introduces each message?

What does `SM` stand for here?  Is there a scheme?

Reply:
I believe it stands for Service Manager.
I'll refactor this to include the errors within a single error file as more will appear soon

Reply:
updated